src/inc/CMakeLists.txt

```diff
@@ -50,23 +50,22 @@ add_compile_options(/TC)
 
 else()
 
-#The MIDL tool exists for Windows only, so for other systems, we have the prebuilt xxx_i.c files checked in
+#The MIDL tool exists for Windows only, so for other systems, we have the prebuilt xxx_i.cpp files checked in
 
 # The prebuilt files contain extra '!_MIDL_USE_GUIDDEF_' after the #endif, but not in the comment.
 # In order to not to have to modify these prebuilt files, we disable the extra tokens warning.
 add_compile_options(-Wno-extra-tokens)
-
+add_compile_options(-D_MIDL_USE_GUIDDEF_)
```

Comment:
There was undefind reference error when linking executable crossgen:

> Linking CXX executable crossgen
/home/kchoi/coreclr/src/pal/inc/rt/palrt.h:326: error: undefined reference to 'IID_ICorSvcLogger'
/home/kchoi/coreclr/src/vm/ceeload.cpp:4636: error: undefined reference to 'IID_ISymUnmanagedWriter3'


            // manipulate the writer.
            SafeComHolderPreemp<ISymUnmanagedWriter3> pWriter3;
          > HRESULT thr = pWriter->QueryInterface(IID_ISymUnmanagedWriter3, (void**)&pWriter3);
            CONSISTENCY_CHECK(SUCCEEDED(thr));
            if (SUCCEEDED(thr))